osquery/killswitch/killswitch_plugin.cpp

```diff
@@ -0,0 +1,89 @@
+#include <string>
+
+#include <chrono>
+#include <osquery/flags.h>
+#include <osquery/killswitch.h>
+#include <osquery/killswitch/killswitch_plugin.h>
+#include <osquery/registry_factory.h>
+
+namespace osquery {
+
+CREATE_REGISTRY(KillswitchPlugin, "killswitch");
+
+Expected<std::map<std::string, bool>, KillswitchPlugin::ParseMapJSONError>
```

Comment:
There is only one difference between `std::map` and `std::unordered_map` - the defined order (usually sorted) in `std::map`. If you don't care about order of elements in your `map` it is better to use `unordered_map` instead. It is faster and even a bit more cache friendly. What do you think?

Reply:
Just my 2-cents on this -- I'd always optimize for speed or perf of execution, and then secondary on volatile memory footprint. That's only because we have expectation that osquery be a very fast and performant agent and don't make as strong of guarantees on how much memory it's using (not always ;)). That being said, if it's expected this will be a smaller sized set not sure it's a huge deal?

Reply:
2 reviewer is the power <3 changed it to unordered_map